File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/03. Movie Destination/03. Movie Destination.py
budget = float(input())
destination = input()
season = input()
day_count = int(input())

country_dict = {
    'Dubai' : {'Winter' : 45000, 'Summer' : 40000,},
    'Sofia' : {'Winter' : 17000, 'Summer' : 12500,},
    'London' : {'Winter': 24000, 'Summer' : 20250,},
}

total_sum = country_dict[destination][season] * day_count

if destination == 'Dubai':
    total_sum *= 0.70
elif destination == 'Sofia':
    total_sum *= 1.25

if budget < total_sum:
    print(f'The director needs {total_sum - budget:.2f} leva more!')
else:
    print(f'The budget for the movie is enough! We have {budget - total_sum:.2f} leva left!')


'''
1000000
Dubai
Summer
5
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/03. Movie Destination/03. Movie Destination.js
function  movieDestination(budget, destination, season, dayCount) {
    const country_dict = {
        'Dubai' : {'Winter' : 45000, 'Summer' : 40000,},
        'Sofia' : {'Winter' : 17000, 'Summer' : 12500,},
        'London' : {'Winter': 24000, 'Summer' : 20250,},
    }

    let totalSum = country_dict[destination][season] * dayCount

    if (destination == 'Dubai'){
        totalSum *= 0.70
    }else if(destination == 'Sofia'){
        totalSum *= 1.25
    }

    if (budget < totalSum)
        return `The director needs ${(totalSum - budget).toFixed(2)} leva more!`
    return `The budget for the movie is enough! We have ${(budget - totalSum).toFixed(2)} leva left!`
}

// console.log(movieDestination(400000, 'Sofia', 'Winter', 20))
